airbyte-integrations/template/python-source/README.md

```diff
@@ -1,3 +1,32 @@
+### Create a Python Airbyte Source
+1. If you are trying to create an Airbyte Source from a Singer Tap, you should follow [this guide](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/template/singer-source/README.md) instead.
+1. Copy the template into a directory with your integration name. `cp -r airbyte-integrations/template/python-source airbyte-integrations/source-SOURCENAME`.
+1. ```
+   mv airbyte-integrations/source-SOURCENAME/template_python_source airbyte-integrations/source-SOURCENAME/SOURCENAME_source
+   ``` 
+1. Update the name, description, and dependencies in `setup.py`
+    1. Note that release dependencies should go in `setup.py`, NOT `requirements.txt`. The requirements file is only used for local development. 
```

Comment:
dumb question: if i have a 3rd party dep i need, do i need to put it in setup.py AND requirements.txt or is just putting it setup.py enough?

Reply:
Updated to make it clear that all of your dependencies should go in setup.py and that requirements.txt is reserved for internal Airbyte dependencies.